requirements.txt

```diff
@@ -1,12 +1,16 @@
+antlr4-python3-runtime==4.7.2
 numpy>=1.17.4
 scipy>=1.0.0
 sympy>=1.5
 networkx>=2.0
-quantum-blackbird>=0.2.0
+quantum-blackbird==0.2.3
+python-dateutil==2.8.0
 thewalrus>=0.11
 numba>=0.48.0
 toml
 appdirs
+requests==2.22.0
+urllib3==1.25.3
 tensorflow==1.3
 tensorflow-tensorboard>=0.1.8
```

Comment:
If we decide to remove TF requirements as well, then this would neeed changing :)

Reply:
Yep, will be changed once the `tf2` branch is merged in